checker/src/main/java/org/checkerframework/checker/signedness/SignednessAnnotatedTypeFactory.java

```diff
@@ -74,11 +73,73 @@ protected void addComputedTypeAnnotations(
         // but adding the top type to them, which permits flow-sensitive type refinement.
         // (When it is possible to default types based on their TypeKinds,
         // this whole method will no longer be needed.)
+        addSignednessGlbAnnotation(tree, type);
         addUnknownSignednessToSomeLocals(tree, type);
 
         super.addComputedTypeAnnotations(tree, type, iUseFlow);
     }
 
+    /**
+     * Refines the type of an expression such as array, method invocation or an integer primitive
```

Comment:
If it's any expression, there is no need to give examples.  It's any expression of primitive type, I think.